pkgs/development/compilers/llvm/12/default.nix

```diff
@@ -134,11 +134,23 @@ let
       inherit (libraries) libunwind;
     };
 
-    lldb = callPackage ./lldb {
-      inherit llvm_meta;
-      inherit (darwin) libobjc bootstrap_cmds;
-      inherit (darwin.apple_sdk.libs) xpc;
-      inherit (darwin.apple_sdk.frameworks) Foundation Carbon Cocoa;
```

Comment:
Changes the override interface of llvmPackages_12.lldb, but probably doesn't matter.